src/renderElements.js

```diff
@@ -29,7 +28,21 @@ export function renderLoader() {
     bodyContainerDiv.insertBefore(loader, gridContainer);
 }
 
-export function removeLoader() {
-    const loader = document.getElementsByClassName("loader").item(0);
-    loader.remove();
-}
\ No newline at end of file
+export function renderPagination(currentPage, maxPage) {
+
+    const pageButtonsTop = makePageButtons(currentPage, maxPage);
+    const pageButtonsBottom = makePageButtons(currentPage, maxPage);
```

Comment:
If they are the same thing, why should we treat them as separate things?

Reply:
My intent here was to render the same numeric buttons on two different parts of the webpage. One inside the top pagination div and the other one inside the bottom pagination div. The first thing I tried was this:

```
const pageNumericButtons = makePageButtons(currentPage, maxPage);

pageNumericButtons.forEach(button => {
        paginationDivs.item(0).insertBefore(button, nextPageButtons.item(0));
        paginationDivs.item(1).insertBefore(button, nextPageButtons.item(1));
    });
```

But the problem is that for some reason, this method only inserts the buttons on the last paginationDiv described. The code above results in the following structure:

![image](https://user-images.githubusercontent.com/80173402/120519212-c0b5e400-c3a8-11eb-9676-dbddf230ec17.png)

If we change the order of the paginationDivs, it also changes where the buttons are inserted:
```
pageNumericButtons.forEach(button => {
        paginationDivs.item(1).insertBefore(button, nextPageButtons.item(1));
        paginationDivs.item(0).insertBefore(button, nextPageButtons.item(0));
    });
```
Resulting:

![image](https://user-images.githubusercontent.com/80173402/120519687-48035780-c3a9-11eb-85c7-61a85a31c80e.png)

Reply:
Oh I think I found a solution that suits better than what I did previously. Using the method `.cloneNode` solves the problem:
```
const pageNumericButtons = makePageButtons(currentPage, maxPage);

pageNumericButtons.forEach(button => {
        paginationDivs.item(0).insertBefore(button, nextPageButtons.item(0));
        paginationDivs.item(1).insertBefore(button.cloneNode(true), nextPageButtons.item(1));
    });
```